plenum/test/view_change/test_6th_node_join_after_view_change_by_primary_restart.py

```diff
@@ -117,7 +117,7 @@ def test_6th_node_join_after_view_change_by_master_restart(
 
         timeout = waits.expectedPoolCatchupTime(nodeCount=len(pool_of_nodes))
         for node in pool_of_nodes:
-            looper.run(eventually(catchuped, node, timeout=2 * timeout))
+            looper.run(eventually(catchuped, node, timeout=3 * timeout))
```

Comment:
Are you sure we need to increase it? Could you describe a problem?

Reply:
Sometimes viewchange does not complete in time. So we need to wait longer.